docling_core/types/doc/document.py

```diff
@@ -1518,6 +1699,179 @@ def export_to_text(  # noqa: C901
             image_placeholder="",
         )
 
+    def export_to_html(  # noqa: C901
+        self,
+        from_element: int = 0,
+        to_element: int = sys.maxsize,
+        labels: set[DocItemLabel] = DEFAULT_EXPORT_LABELS,
+        image_dir: Optional[Path] = None,
```

Comment:
Same comment as for `export_to_dict`, we cannot simply have an `image_dir` argument in here.